con-master/src/main/java/ba/codecentric/master/controller/RoomController.java

```diff
@@ -18,7 +17,7 @@ public RoomController(RoomService roomService) {
         this.roomService = roomService;
     }
 
-    @RequestMapping(value = "/room", method = RequestMethod.POST)
+    @PostMapping(value = "/rooms")
```

Comment:
If we are using here @PostMapping, then we need use newer annotations on all other controllers. So, please do it for all controllers.